backend/dgraph/Cargo.toml

```diff
@@ -7,6 +7,9 @@ edition = "2018"
 doctest = false
 
 [dependencies]
-serde = {version = "1.0.126", features = ["derive"]}
-tokio = {version = "1", features = ["full"]}
+serde = { version = "1.0.126", features = ["derive"] }
+tokio = { version = "1", features = ["full"] }
 gql_client = "1.0.7"
+
+[features]
```

Comment:
```suggestion
[features]
default = ["dgraph-tests"]
```

Reply:
This is basically because I feel like it would be good to run our dgraph tests regularly as devs, so would rather disable them in CI rather than have to enable the every time in our IDE.
Although we can set features on for a VS code workspace so I'm happy if you leave it as is.

Reply:
Nah nice I like this!